__tests__/task.test.js

```diff
@@ -131,6 +155,8 @@ describe('Task', () => {
       expect(statusCode).toBe(302);
       const tasks = await app.objection.models.task.query();
       expect(tasks).toHaveLength(0);
+      const labels = await existingTask.$relatedQuery('labels');
+      expect(labels).toHaveLength(0);
```

Comment:
maybe also check that label still exists in database?

Reply:
done